src/main/java/org/broadinstitute/hellbender/tools/copynumber/DetermineGermlineContigPloidy.java

```diff
@@ -28,73 +28,113 @@
+ * is sex determination, as well as informing about expected SNP allele frequencies and copy-ratio plateaus. This
+ * tool uses counts data for germline karyotyping. </p>
+ *
+ * <p>Performing germline karyotyping using counts data requires calibrating ("modeling") the technical coverage bias
+ * and variance for each contig. The Bayesian model and the associated inference scheme implemented in
+ * {@link DetermineGermlineContigPloidy} includes provisions for inferring and explaining away much of the technical
+ * variation. Furthermore, karyotyping confidence is automatically adjusted for individual samples and contigs.</p>
+ *
+ * <p>Running {@link DetermineGermlineContigPloidy} is the first computational step in the GATK germline CNV calling
+ * pipeline. It provides a baseline ("default") copy-number state for each contig/sample with respect to which the
+ * probability of alternative states is allocated.</p>
+ *
+ * <h3>Python environment setup</h3>
+ *
+ * <p>The computation done by this tool, aside from input data parsing and validation, is performed outside of the Java
+ * Virtual Machine and using the <em>gCNV computational python module</em> <code>"gcnvkernel"</code>. It is crucial that
+ * the user has properly set up a python <code>conda</code> environment with <code>gcnvkernel</code> and its dependencies
+ * installed. If the user intends to run {@link DetermineGermlineContigPloidy} using one the official GATK Docker images,
+ * the python environment is already set. Otherwise, the environment must be created and activated as described in the
+ * main GATK README.md file.</p>
+ *
+ * <h3>Tool run modes</h3>
+ *
+ * <p>This tool has two operation modes as described below:</p>
  * <dl>
  *     <dt>COHORT mode:</dt>
  *
- *     <dd>If a ploidy-model directory is not provided via the {@code model} argument, the tool is run in the COHORT
- *      mode. In this mode, ploidy-model parameters (e.g. coverage bias and variance for each contig) are inferred,
- *      along with baseline contig ploidy states of each sample. A tab-separated table specifying prior probabilities
- *      for each ploidy state and for each contig is required in this mode and must be specified via the
- *      {@code contig-ploidy-priors} argument. The following shows an example of such a table:
- *      <br>
- *      <table border="1" width="80%">
- *          <tr>
- *              <td>CONTIG_NAME</td> <td>PLOIDY_PRIOR_0</td> <td>PLOIDY_PRIOR_1</td> <td>PLOIDY_PRIOR_2</td> <td>PLOIDY_PRIOR_3</td>
- *          </tr>
- *          <tr>
- *             <td>1</td> <td>0.01</td> <td>0.01</td> <td>0.97</td> <td>0.01</td>
- *          </tr>
- *          <tr>
- *             <td>2</td> <td>0.01</td> <td>0.01</td> <td>0.97</td> <td>0.01</td>
- *          </tr>
- *          <tr>
+ *     <dd>If a ploidy model parameter directory is not provided via the <code>--model &lt;path&gt;</code> argument,
```

Comment:
Elsewhere in the CNV Javadocs we simply use `{@code argument-name}` to refer to arguments, rather than `<code>--argument-namel &lt;argument-type&gt;</code>`.  I'm fine with adding `--` as long as we are consistent and add it elsewhere; adding the argument type is a bit more idiosyncratic (and slightly confusing, in my opinion).  Will defer to @sooheelee if she has any strong opinions, otherwise I'd just leave it as it was.

Reply:
Changed all instances to `{@code argument-name}`.